Tests/PlayMode/EditMediatorTest.cs

```diff
@@ -637,18 +644,18 @@ public IEnumerator ShouldRequestChangeMaterialOfObjects_When_ReceivesChangeMater
     {
         JObject material = new JObject()
         {
-            { "id", 1 },
+            { "id", 123456 },
             { "texture", "material-1-url" },
             { "widthMilimeters", 2000 },
-            { "heightMilimeters", 1500}
+            { "heightMilimeters", 1500 }
         };
-        Dictionary<string, object> message = new Dictionary<string, object>
+        JObject message = new JObject()
         {
             { "type", WebMessageType.changeObjectsMaterial },
-            { "payload", new Dictionary<string, object>
+            { "payload", new JObject()
                 {
-                    {"objectIds", new string[] { "id-0", "id-1" } },
-                    { "material", material },
+                    { "objectIds", new JArray(new string[] { "id-0", "id-1" }) },
+                    { "material", material }
```

Comment:
I like to place commas when possible so in case we need to add more properties to the objects, we don't need to change an old line just to put a comma. 
```suggestion
                    { "material", material },
```